website/src/components/sidebar/sidebar.tsx

```diff
@@ -97,8 +97,10 @@ const MainNavLink = ({ href, icon, children }) => {
         fontWeight="semibold"
         transitionProperty="colors"
         transitionDuration="200ms"
-        color={active ? "gray.900" : "gray.500"}
-        _hover={{ color: "gray.900" }}
+        color={
+          active ? useColorModeValue("gray.900", "whiteAlpha.900") : "gray.500"
```

Comment:
It looks like your are violating the [Only Call Hooks at the Top Level](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level) rule here. You are using `useColorModeValue` (which in turn uses `useContext`) in a loop.

Reply:
Ah, yes that's correct. Thank you. Fixing.